app/src/main/java/com/internship/move/feature/scooter_unlock/endRideDTO.kt

```diff
@@ -0,0 +1,17 @@
+package com.internship.move.feature.scooter_unlock
+
+import android.os.Parcelable
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
```

Comment:
The `RideHistoryItemDTO` comment applies here too.
Also, you have a typo in the name, start class names with capital letters :P